gulpfile.js

```diff
@@ -138,6 +141,31 @@ function serverServe(done) {
 }
 
 
+//
+// Installation of components/dependencies
+//
+
+// Copy Font Awesome 4 files
```

Comment:
Not necessary for this PR, but it might be worth upgrading to FA5 for v6 while we're doing this type of work.

Reply:
Agreed. I'll add that on the list for a future PR.